api/types.go

```diff
@@ -0,0 +1,47 @@
+package api
+
+type Context struct {
+	State DebuggerState `json:"state"`
```

Comment:
I think it would be best at this point to hash out exactly what the types will be. If we agree on the conceptual model (REST), everything will follow from the API types.